src/registry_schemas/schemas/registration.json

```diff
@@ -32,6 +32,14 @@
                 "contactPoint": {
                     "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/contactPoint"
                 },
+                "naicsCode": {
```

Comment:
This doesn't follow the pattern of a block with all the info in it.
NAICS will also get used across all business types, so should be an easier block to add and maybe in the business block?

Reply:
moved it